bitnami/harbor/templates/core/core-dpl.yaml

```diff
@@ -0,0 +1,148 @@
+          failureThreshold: {{ .Values.core.readinessProbe.failureThreshold }}
+        {{- end }}
+        envFrom:
+        - configMapRef:
+            name: "{{ template "harbor.core" . }}-envvars"
+        - secretRef:
+            name: "{{ template "harbor.core" . }}-envvars"
+        env:
+        {{- if .Values.coreImage.debug }}
+        - name: BITNAMI_DEBUG
+          value: "true"
+        {{- end }}
+          - name: CORE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "harbor.core" . }}
+                key: secret
+          - name: JOBSERVICE_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: "{{ template "harbor.jobservice" . }}"
+                key: secret
+        ports:
+        - containerPort: 8080
+          name: http
+        volumeMounts:
+        - name: config
+          mountPath: /etc/core/app.conf
+          subPath: app.conf
+        - name: secret-key
+          mountPath: /etc/core/key
+          subPath: key
+        - name: token-service-private-key
+          mountPath: /etc/core/private_key.pem
+          subPath: tls.key
+        {{- if .Values.service.tls.enabled }}
+        - name: ca-download
+          mountPath: /etc/core/ca/ca.crt
+          subPath: ca.crt
+        {{- end }}
+        - name: psc
+          mountPath: /etc/core/token
+        {{- if .Values.core.resources }}
+        resources:
+          {{ toYaml .Values.core.resources | nindent 10 }}
+        {{- end }}
+      volumes:
+      - name: config
+        configMap:
+          name: {{ template "harbor.core" . }}
+      - name: secret-key
+        secret:
+          secretName: {{ template "harbor.core" . }}
+          items:
+            - key: secretKey
+              path: key
+      - name: token-service-private-key
+        secret:
+          {{- if .Values.core.secretName }}
+          secretName: {{ .Values.core.secretName }}
+          {{- else }}
+          secretName: {{ template "harbor.core" . }}
+          {{- end }}
+      {{- if .Values.service.tls.enabled }}
+      - name: ca-download
+        secret:
+        {{- if eq (include "harbor.autoGenCertForIngress" .) "true" }}
+          secretName: "{{ template "harbor.ingress" . }}"
+        {{- else if eq (include "harbor.autoGenCertForNginx" .) "true" }}
+          secretName: {{ template "harbor.nginx" . }}
+        {{- else }}
+          secretName: {{ .Values.service.tls.secretName }}
+        {{- end }}
+          items:
+            - key: ca.crt
+              path: ca.crt
+      {{- end }}
+      - name: psc
+        emptyDir: {}
+      {{- with .Values.core.nodeSelector }}
+      nodeSelector:
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.core.affinity }}
+      affinity:
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.core.tolerations }}
+      tolerations:
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
**nodeSelector**, **tolerations** and **affinity** should be evaluated as a template:
```yaml
      {{- with .Values.nodeSelector }}
      nodeSelector: {{ tpl (toYaml .) $ | nindent 8 }}
      {{- end }}
      {{- with .Values.tolerations }}
      tolerations: {{ tpl (toYaml .) $ | nindent 8 }}
      {{- end }}
      {{- with .Values.affinity }}
      affinity: {{ tpl (toYaml .) $ | nindent 8 }}
      {{- end }}
```

Remember to update these parameters instructions in the **README.md** and **values.yaml** adding the message:
```
The value is evaluated as a template.
```

Reply:
why?

Reply:
I could include labels with references to values such as `{{ .Release.name }}`